nixos/modules/programs/foot/default.nix

```
@@ -0,0 +1,79 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+
+let
+  cfg = config.programs.foot;
+
+  settingsFormat = pkgs.formats.iniWithGlobalSection { listsAsDuplicateKeys = true; };
```

FYI, `mkKeyValue` can be set here

```suggestion
  settingsFormat = pkgs.formats.iniWithGlobalSection {
    listsAsDuplicateKeys = true;
    mkKeyValue = ...
  };
```

---

Good catch!